invenio_app_ils/circulation/utils.py

```diff
@@ -82,3 +82,8 @@ def circulation_overdue_loan_days(loan):
     """Return the amount of days a loan is overdue."""
     end_date = arrow.get(loan["end_date"])
     return (arrow.get().utcnow() - end_date).days
+
+
+def circulation_upcoming_return_range():
+    """Return a default loan duration in timedelta."""
```

Comment:
true, thanks

Reply:
I might have changed my mind, can we make this easily configurable? So this will be just a configuration var? WDYT?
Actually can you create an issue that we need:
* send an e-mail saying that "the loan is expiring in 7 days" on the upcoming configured date
* send an e-mail saying that "the loan has expired today" in the day of expiration